src/debugger/nodeDebugWrapper.ts

```diff
@@ -135,8 +136,16 @@ Telemetry.init("react-native-debug-adapter", version, true).then(() => {
             args.target || "simulator"
         ];
 
+        if (!isNullOrUndefined(args.logCatArguments)) { // We add the parameter if it's defined (adapter crashes otherwise)
+            args.args = args.args.concat([args.logCatArguments]);

```

Comment:
Perhaps we should JSON.stringify and JSON.parse? I believe that should work out as long as the options aren't very large.

This is another issue that will go away once we move more stuff out of the debugee process though.